myapp/stock_api.py

```diff
@@ -46,4 +46,8 @@ def get_stock_info(symbol):
 
 
 def get_stock_historic_prices(symbol, time_range='1m'):
-	return _request_data('/stable/stock/{symbol}/chart/{time_range}'.format(symbol=symbol, time_range=time_range))
\ No newline at end of file
+	return _request_data('/stable/stock/{symbol}/chart/{time_range}'.format(symbol=symbol, time_range=time_range))

```

Comment:
you really should be using f strings 
writing this once, but relevant for all of you code

Reply:
This change should not be in the pull request

Reply:
Fixed.